index.html

```diff
@@ -10,16 +10,170 @@
     <script src="app.js" defer></script>
 </head>
 
-<body class="center" style="background-color: #f8f8f8;">
-    <header style="background-color: white;">
-        <nav style="display: flex; align-items: center;">
-            <h1>Meditation Summit</h1>
-            <ul style="display: flex;">
-                <li><a href="#home">Home</a></li>
-                <li><a href="#about">About</a></li>
+<body>
+    <!-- mobile nav -->
+    <header class="mobile">
+        <nav class="topNav">
+            <button class="ham">
+                <img src="./assets/ham.svg" alt="menu">
+            </button>
+        </nav>
+        <nav class="sideNav">
+            <button class="menuX">
+                <img src="./assets/x.svg" alt="cancel">
+            </button>
+            <ul>
+                <li><a class="menuLink" href="#home">Home</a></li>
+                <li><a class="menuLink" href="#program">Program</a></li>
+                <li><a class="menuLink" href="#teachers">Teachers</a></li>
+                <li><a class="menuLink" href="#partners">Partners</a></li>
+                <li><a class="menuLink" href="#about">About</a></li>
             </ul>
         </nav>
     </header>
+
+    <!-- desktop-nav -->
+    <nav class="flex items-center justify-around desktop desktop-nav">
+        <h1><a href="#home">Meditation Summit</a></h1>
+        <ul>
+            <li><a href="#home">Home</a></li>
+            <li><a href="#program">Program</a></li>
+            <li><a href="#teachers">Teachers</a></li>
+            <li><a href="#partners">Partners</a></li>
+            <li><a href="#about">About</a></li>
+        </ul>
+    </nav>
+    <section id="home" class="home">
+        <div class="primary-color home-text-1">Champions! Welcome</div>
+        <div class="primary-color home-text-2">
+            THE GOLDEN ORDER<br>GLOBAL SUMMIT<br>2022</div>
+        <div class="home-text-3">
+            A joyful celebration believing in the value of spells, incantations and ashes of war, creating a positive
+            change with champions from over 80 nations is taking place in October, in the Lands Between.</div>
+        <div class="home-text-4">2022.10.15(THU) ~ 16(FRI)
+        </div>
+        <div class="home-text-5">@ Academy of Raya Lucaria,
+            Caria Manor and more</div>
+    </section>
+    <section id="program" class="flex justify-center text-center program">
+        <div>
+            <header>
+                <div>
+                    Main Program
+                </div>
+                <div class="flex justify-center">
+                    <div class="divider"></div>
```

Comment:
- [ ] Please make sure to fix the alignment of elements of this section on the mobile screens and It should match with the template design. (The card's elements are aligned in rows for mobile screens).

### Your design
![image](https://user-images.githubusercontent.com/83803414/174878926-58aa8c8b-430a-439b-a09b-e60802006b43.png)

### Behance design

![image](https://user-images.githubusercontent.com/83803414/174878827-db37088e-7c65-4e47-891d-505bdad69c31.png)